### ISSUE TITLE
Upgrading link leads to 404

### ISSUE THREAD

**Issue by pat-s** [meilisearch/meilisearch-kubernetes#167]
The upgrading link shown leads to a 404

```
│ Error: Your database version (1.0.0) is incompatible with your current engine version (1.1.1).                                                                   │
│ To migrate data between Meilisearch versions, please follow our guide on https://docs.meilisearch.com/learn/update_and_migration/updating.html. 
```

https://www.meilisearch.com/docs/learn/update_and_migration/updating.html.

**Comment by alallema** [comment]
Hi @pat-s,
Thanks for bringing this up, but it doesn't belong to this repo but to the engine repo, if I'm not mistaken.
Let me transfer it there.

**Comment by pat-s** [comment]
The link I posted in OP leads to

![image](https://user-images.githubusercontent.com/8430564/234099508-fe7ac866-465c-4aa6-818b-f561c74db200.png)

**Comment by alallema** [comment]
> The link I posted in OP leads to

Yes, I can see that, but where did you find it? Because it is not coming from the error message, does it?
This way I know where we have to fix it because I'm not sure this one should work

**Comment by pat-s** [comment]
It comes from the link posted in OP.

**Comment by brunoocasali** [comment]
Actually @pat-s, if we use the link coming from the error message:

> Error: Your database version (1.0.0) is incompatible with your current engine version (1.1.1).                                                                   
 To migrate data between Meilisearch versions, please follow our guide on https://docs.meilisearch.com/learn/update_and_migration/updating.html. 

It does redirect.

**Comment by pat-s** [comment]
The following does not: https://www.meilisearch.com/docs/learn/update_and_migration/updating.html

and my browser redirects to the `www` subdomain first when I click on it.

**Comment by brunoocasali** [comment]
> The following does not: https://www.meilisearch.com/docs/learn/update_and_migration/updating.html
> 
> and my browser redirects to the `www` subdomain first when I click on it.

Yes, but that's the thing we do not understand each other. I know this link: `https://www.meilisearch.com/docs/learn/update_and_migration/updating.html` does not work, but where does it come from?

**Comment by pat-s** [comment]
Ah! 

I get redirected from https://docs.meilisearch.com/learn/update_and_migration/updating.html -> https://www.meilisearch.com/docs/learn/update_and_migration/updating.html when clicking on the URL in my browser (Chromium 112.0.5615.137).

**Comment by pat-s** [comment]
Ahh I think I've spotted the issue. In the OP you see that the `.` gets appended to the URL and then some weird redirection happens. If I just click the link in my terminal, the dot gets appended and you almost don't notice that. 

Maybe it's safer remove the period at the end of the sentence. Not sure if it's a terminal issue or something else but I guess the safest would be to remove the dot 😃️

404: `https://docs.meilisearch.com/learn/update_and_migration/updating.html.`
200: `https://docs.meilisearch.com/learn/update_and_migration/updating.html`

But could it be that you also changes something? Or maybe it was my cache but usually I try something like this in private windows. But can't recall anymore...

**Comment by alallema** [comment]
In fact if you look carefully it has even a `.html` at the end that is redirected without, so if you copy with the dot it will not work and redirect to: https://www.meilisearch.com/docs/learn/update_and_migration/updating
Thanks for closing this
